po/common/en.po

```diff
@@ -5130,68 +5130,3 @@ msgstr "Products that do not match your preferences"
 msgctxt "products_match_unknown"
 msgid "Products for which we currently miss data to determine if they match your preferences"
 msgstr "Products for which we currently miss data to determine if they match your preferences"
-
-msgctxt "forest_footprint"
```

Comment:
why are those strings removed?

Reply:
because of the discrepancies between common.pot and en.po